test/e2e_tests/data_validation_test.go

```diff
@@ -181,6 +197,77 @@ func (suite *dataValidationTestSuite) TestFileUpdate() {
 	suite.dataValidationTestCleanup([]string{localFilePath, remoteFilePath, suite.testCachePath})
 }
 
+func validateData(fileName string, fileSize string, suite *dataValidationTestSuite) {
+	defer wg.Done()
+
+	localFilePath := suite.testLocalPath + "/" + fileName
+	remoteFilePath := suite.testMntPath + "/" + fileName
+	fmt.Println("Local file path: " + localFilePath)
+
+	// create the file in local directory
+	srcFile, err := os.OpenFile(localFilePath, os.O_CREATE, 0777)
+	suite.Equal(nil, err)
+	srcFile.Close()
+
+	// write to file in the local directory
+	var fileBuff []byte
+	if fileSize == "large" {
+		fileBuff = make([]byte, (500 * 1024 * 1024))
```

Comment:
can we add some test case for 2-3 files which are 1/2gb in size.

Reply:
Done